spec/domain/output_document_spec.rb

```diff
@@ -129,8 +129,10 @@ def excludes_all_circumstances
     end
   end
 
-  describe '#greeting' do
-    it 'needs details'
+  describe '#lead' do
+    subject { output_document.lead }
+
+    it { is_expected.to eq('You recently had a Pension Wise guidance appointment with A Guider from The Pensions Advisory Service on March 30, 2015') }
```

Comment:
"A Guider" isn't representative test data - it should be a first name.

The date should be formatted as "30 March 2015" as per the GOV.UK style guide.